examples/image_annotate/definitions/unannotated_imagesets.sqlx

```diff
@@ -0,0 +1,19 @@
+config {
+    type: "view"
+}
+
+js {
+    // Ref is avoided here to avoid the circular dependency chain.
+    const ANNOTATED_IMAGESETS = `${ctx.schema()}` + ".annotated_imagesets"
+}
+
+SELECT
+  *
+FROM
+  ${ref("all_imagesets")}
+LEFT JOIN
```

Comment:
`WHERE uri NOT IN (SELECT uri FROM ${ANNOTATED_IMAGESETS})` is probably easier to read. Not sure if more or less efficient though